### ISSUE TITLE
Wrong handle position with reversed scroll

### ISSUE THREAD

**Issue by mc-petry** [KingSora/OverlayScrollbars#625]
**Describe the bug**
I have implemented a chat interface where messages flow from bottom to top, with the scroll bar anchored to the bottom. To achieve this, I used a native CSS solution, applying flex-direction in reverse to the container. However, I encountered an issue where the scrollbar handle incorrectly jumps to the bottom when scrolling to the top of the chat.

```
[data-overlayscrollbars-contents] {
  display: flex;
  flex-direction: column-reverse;
}
```
Seems everything works well except scroll handle at top position.

**To Reproduce**

1. Go to [StackBlitz](https://stackblitz.com/edit/vitejs-vite-mih3nq?file=src%2FApp.tsx)
2. Scroll to top
3. See error: scroll handle jumped to the bottom

**Expected behavior**
The scrollbar handle should remain at the top when the user scrolls to the top.

**Examples**
[StackBlitz](https://stackblitz.com/edit/vitejs-vite-mih3nq?file=src%2FApp.tsx)

**Environment**

- Used Operating System(s): MacOS 14.3.1
- Used Browser(s) (with version): Chrome 123.0.6312.107 (Official Build) (arm64)

**Comment by KingSora** [comment]
Good day @mc-petry :)

`OverlayScrollbars` doesn't track the flow direction and makes it calculations always based on the default flow direction. (except for `direction: rtl`)

Please wrap your content with an additional `div` and apply the styles there: https://stackblitz.com/edit/vitejs-vite-l5egrt?file=src%2FApp.tsx

**Comment by mc-petry** [comment]
Thank @KingSora for your quick reply.
I updated [example](https://stackblitz.com/edit/vitejs-vite-mih3nq?file=src%2FApp.tsx).
Try to click "add message".
As you can see, at the first approach, the scroll sticked to the bottom.
In second one with inner container - no.

**Comment by KingSora** [comment]
I see... Well, I have to think about how to fix this issue, because the browser currently doesn't expose any hints about what the min. and max. scroll position in one direction can be. It can be calculated in the range `0..+x` but having a reverse flow (column or row) or a `direction: rtl` will change that range to `0..-x` which I can't detect except when I track those properties manually

Sorry, but I can't provide a good workaround for the time being..

**Comment by KingSora** [comment]
The more I've worked on a "fix" for this problem the more I realized that it is a whole new feature. I've just released `v2.8.0` which includes this feature. I've updated your example: https://stackblitz.com/edit/vitejs-vite-75rxhi

**Comment by mc-petry** [comment]
> The more I've worked on a "fix" for this problem the more I realized that it is a whole new feature. I've just released `v2.8.0` which includes this feature. I've updated the version of your example to `v2.8.0`: [stackblitz.com/edit/vitejs-vite-75rxhi](https://stackblitz.com/edit/vitejs-vite-75rxhi)

Thanks a lot, works like a charm!

**Comment by bstaley** [comment]
I've noticed that this fails sometimes. When opening https://stackblitz.com/edit/vitejs-vite-75rxhi the native bar is at the top and it forces you to drag the thumb in opposite(to standard) directions. However, if you refresh the example it will bounce between scrolling correctly and incorrectly.

I'm taking a very similar approach to this issue to show messaging and having the same issue. The only difference is my selector is `[overlayscrollbars]` and I'm on Angular 21 / overlayscrollbars 2.13.0. Any ideas?
